recipe/meta.yaml

```diff
@@ -54,12 +54,12 @@ requirements:
     - tbb  >=2021.0
```

Comment:
@stuartarchibald wdyt? IOW just pin during the build, but relax during runtime. This won't require a hot-fix. If we do find it has issues with 2022, we can pin here and hot-fix after